locales/en.json

```
@@ -933,6 +933,7 @@
       "time": "Time"
     },
     "noResults": "Looks Like There's Nothing Here Yet",
+    "noFlips": " Looks like it's quiet in here!",
```

I would remove exclamation mark
```suggestion
    "noFlips": "Looks like it's quiet in here",
```